ballerina/tests/from_xml_test.bal

```diff
@@ -899,3 +899,140 @@ isolated function testFromXmlWithEmpty5() returns error? {
     test:assertEquals(rec, output, msg = rec.toString());
     test:assertEquals((), rec.Catering?.statusCode);
 }
+
+@Namespace {
+    prefix: "ns",
+    uri: "http://sdf.com"
+}
+type Test record {
+    Test1 xs\:id;
+};
+
+
+type Test1 record {
+
+};
+
+@test:Config {
+    groups: ["fromXml"]
+}
+isolated function testFromXmlXXXX() returns error? {
+    xml data = xml `<ns:Test xmlns="example.com"
+                        xmlns:ns="http://sdf.com" status="paid">?
+                        <xs:id xmlns:xs="http://sdf.com">1</xs:id>
+                    </ns:Test>`;
+    Test output = {
+        xs\:id:{"#content": "1"},
+        "#content": "?"
+    };
+    Test rec = check fromXml(data);
+    test:assertEquals(rec, output, msg = rec.toString());
```

Comment:
is this a negative testcase?

Reply:
No, This is a duplicate test. I have emoved now [b2f9783](https://github.com/ballerina-platform/module-ballerina-xmldata/pull/492/commits/b2f9783a43b592183e72800fdb446cdbf1a84f3a)